react/Header/UserAccount/UserAccount.less

```diff
@@ -65,15 +65,16 @@
   bottom: 0;
 }
 
-.userDisplayName {
+.userName {
   pointer-events: none;
   overflow: hidden;
   text-overflow: ellipsis;
   display: inline-block;
-  max-width: 101px; // ~ 10 characters
+  white-space: nowrap;
+  max-width: 100px; // ~ 10 characters
 
   @media only screen and (min-width: 400px) {
```

Comment:
Might be preferable to use `@responsive-breakpoint` and then a scaling width below that (ie you get somewhere between 10 and 20 characters as you approach 740px)?